### ISSUE TITLE
Bug: wrong error message

### ISSUE THREAD

**Issue by VikingScientist** [TheBB/Geomaker#6]
![Screenshot from 2020-02-03 14-49-24](https://user-images.githubusercontent.com/3500376/73659208-0ddf5400-4696-11ea-9827-f9b885bac908.png)

Requesting to download an area of less than 16 km<sup>2</sup> still gives the error message that this is too big. My current guess is that it won't accept long_thin areas, but anything less than 4km x 4km is fine.

**Comment by TheBB** [comment]
This is because the bounding box in UTM33 axis-aligned coordinates is bigger than the actual polygon, which is axis-aligned in latitude and longitude.

**Comment by VikingScientist** [comment]
Really? Is the difference this great?
![Screenshot from 2020-02-03 15-24-29](https://user-images.githubusercontent.com/3500376/73660846-4b91ac00-4699-11ea-9f79-a8b71ddd58a0.png)

I tried creating a stupidly long thin region which reportedly is less than 6km<sup>2</sup>, and this also failed.

**Comment by VikingScientist** [comment]
Your reasoning makes sense, but we'll still keep it as a bug since the error message is misleading. Potential solutions

* Write out in clear text in the error message that area on right side is approximation due to coordinate transformation and the actual value is something else
* Show the returned polygon on the map by transforming coordinates
* Compute the actual area on the right-hand side and  show the correct used value

**Comment by TheBB** [comment]
I agree it's misleading. Just to be clear though, the area on the right is not an approximation and it is the genuine area of the polygon. Hoydedata.no then computes a bounding box in the UTM33 coordinate system, which will generally be larger. It's this bounding box that must be limited by 16 km2.

**Comment by VikingScientist** [comment]
So the "square" bounding box is given in (lat,lon), which translates to a parlellogram(ish) region in UTM33 which is the one returned from hoydedata.no. This is also the reason for the cut-off no-data regions in the seemingly rotated rectangle which is returned as a tiff file.

You can actually see this distortion with your naked eye if you look at for instance Nordkapp in google earth (lat-lon) vs hoydedata (UTM33) where the  angle of the coastlines go in different directions

![Screenshot from 2020-02-04 09-31-58](https://user-images.githubusercontent.com/3500376/73727451-c6a6a100-4731-11ea-80ea-6d84d57bf343.png)

**Høydedata.no**

![Screenshot from 2020-02-04 09-32-00](https://user-images.githubusercontent.com/3500376/73727472-cefedc00-4731-11ea-8d2d-92acb8ac49f9.png)

**Google maps**

**Comment by VikingScientist** [comment]
If the "preview" button on #43 is implemented, then this will also be fixed
